src/movies.context.tsx

```diff
@@ -25,14 +24,13 @@ const useMoviesContext = () => {
     const {movies, setMovies} = moviesData;
 
     const addMovie = (movie: InnerState) => {
-        debugger
         setMovies(movies => [...movies, movie])
     };
     const deleteMovie = (index: number) => setMovies((movies) => movies.filter((_movie: any, movieIndex: number) => movieIndex !== index));
```

Comment:
NIT: I wouldn't mind to just keep `_` instead of `_movie`, so lots of devs doing that..

Reply:
NIT: Too much of action in one line, a little bit hard to read, would recommend to split. You can move the function to global utils, it kind of looks universal and usefule in future


```
// utils.ts
// as this is util, usage of any here is a good thing
const filterOutPosition = (input: any[], skipIndex: number) => input.filter((_: any, index: number) => skipIndex !== index);

// in component
const deleteMovie = (index: number) => setMovies((movies) => filterOutPosition(movies, index));

```